parsers/to-style-dictionary/to-style-dictionary.spec.ts

```diff
@@ -562,5 +562,35 @@ describe('To Style Dictionary', () => {
           expect(result).toEqual(expectation);
         });
     });
+    it('Font', () => {
+      seeds()
+        .tokens.filter(({ type }) => type === 'font')
+        .map(font => {
+          const tokenClass: StyleDictionaryTokenClass = (<any>TokensClass)['Font'];
+          const [familyName, subFamilyName] = font.name.split('-');
```

Comment:
Should we care about `subFamilyName` being undefined anytime a font name would diverge from current seed?

Reply:
The objective of this function is to generate objects with variable depths depending on the length of the array. Here it's:
```js
{
    [familyName]: {
        [subfamilyName]: ...
    }
}
```
It could be deeper or smaller. So here it's just a use case.